arviz/plots/backends/matplotlib/bpvplot.py

```diff
@@ -86,7 +86,8 @@ def plot_bpv(
         pp_var_name, _, pp_vals = pp_plotters[i]
 
         obs_vals = obs_vals.flatten()
-        pp_vals = pp_vals.reshape(total_pp_samples, -1)
+        if pp_vals.ndim > 2:
+            pp_vals = pp_vals.reshape(total_pp_samples, -1)
```

Comment:
I think this will break the ArviZ shape convention and interpret the draw dimension as the observations.